packages/event-bus/.eslintrc.json

```
@@ -0,0 +1,25 @@
+{
+  "extends": ["../../.eslintrc.json"],
+  "ignorePatterns": ["!**/*"],
+  "overrides": [
+    {
```

Remove the  rule from the  override since it is included in the base  file

```suggestion
{
  "extends": ["../../.eslintrc.json"],
  "overrides": [
    {
      "files": ["*.json"],
      "parser": "jsonc-eslint-parser",
      "rules": {
        "@nx/dependency-checks": "error"
      }
    }
  ]
}
```